index.html

```diff
@@ -148,39 +148,34 @@ <h2>About Us</h2>
             <p>Ruby NZ was incorporated on June 5th 2014, and is Society #2606303.</p>
           </section>
           <hr />
+
           <section id="current-committee">
             <h2>Current Committee</h2>
-            <p>After the vote held at Rails Camp NZ #7 on Feb 24, 2017 the committee was:</p>
+            <p>After the vote held during the AGM held on the Ruby NZ Slack on May 16, 2018 the committee was:</p>
             <ul>
-              <li><strong>President:</strong> <a href="https://twitter.com/raquelxmoss" target="_blank">Raquel Moss</a></li>
-              <li><strong>Secretary:</strong> <a href="https://twitter.com/danielfone" target="_blank">Daniel Fone</a></li>
+              <li><strong>President:</strong> <a href="https://twitter.com/nz_cub3y" target="_blank">Charles Peach</a></li>
+              <li><strong>Secretary:</strong> <a href="https://twitter.com/containsulfite" target="_blank">Mai Nguyen</a></li>
               <li><strong>Treasurer:</strong> <a href="https://twitter.com/steveh7" target="_blank">Steve Hoeksema</a></li>
               <li>
                 <strong>General Members:</strong>
                 <ul>
-                  <li><a href="https://twitter.com/a_wagener" target="_blank">Amanda Wagener</a></li>
-                  <li><a href="https://twitter.com/psdavey" target="_blank">Patrick Davey</a></li>
-                  <li><a href="https://twitter.com/nz_cub3y" target="_blank">Charles Peach</a></li>
-                  <li><a href="https://twitter.com/maximushung" target="_blank">Max Hung</a></li>
+                  <li><a href="https://twitter.com/charlieablettnz" target="_blank">Charlie Ablett</a></li>
+                  <li><a href="https://twitter.com/ur5us" target="_blank">Juri Hahn</a></li>
+                  <li><a href="https://twitter.com/ioquatix" target="_blank">Samuel Williams</a></li>
                 </ul>
               </li>
             </ul>
-
-            <p><strong>Changes Feb 27, 2018</strong></p>
-            <ul>
-              <li><a href="https://twitter.com/raquelxmoss" target="_blank">Raquel Moss</a> <strong>Stepping down as President</strong></li>
-              <li><strong>New President:</strong> <a href="https://twitter.com/nz_cub3y" target="_blank">Charles Peach</a></li>
-            </ul>
-
-            <p>Special thanks to <a href="https://twitter.com/raquelxmoss" target="_blank">Raquel Moss</a> for leading the current committee 2017-2018!</p>
           </section>
           <hr />
           <section id="past-committee">
             <h2>Past Committee Members</h2>
             <ul>
+              <li><a href="https://twitter.com/a_wagener" target="_blank">Amanda Wagener</a> (general member 2017-2018 and 2014-2015, president 2016-2017, secretary 2015-2016)</li>
+              <li><a href="https://twitter.com/danielfone" target="_blank">Daniel Fone</a> (secretary 2016-2018)</li>
```

Comment:
I don't mind either way, but this could be: (secretary 2016-2018, general member 2015)